### DISCUSSION TITLE
PHP Agent Issue in Windows machine

### DISCUSSION THREAD

**Discussion by AjinThankachan** [apache/skywalking#11790] [Q&A] [answered]
Hi,

OAP Version :9.7
PHP Agent :0.7.0
Server : Xampp 
PHP version : 8.0.30

We have tried to install the PHP agent in the Windows machine using the xampp server and its PHP. But we are not able to see any documents and when we tried to follow the procedure, we were not able to see any .dll file as well for the same. Can you please guide us on installing the php agent on a Windows machine.
 Also once we generated the .dll or .so file can we reuse the same instead of compiling the same every time. 

We appreciate your response and thanks in advance.

Thanks

**Comment by heyanlong** [comment, marked as answer]
No windows support for now
